app/src/main/java/org/pesmypetcare/mypetcare/services/pet/PetManagerAdapter.java

```diff
@@ -203,8 +203,7 @@ public void deletePetsFromUser(User user) {
         ExecutorService executorService = Executors.newSingleThreadExecutor();
         executorService.execute(() -> {
             try {
-                userPets.set(ServiceLocator.getInstance().getPetManagerClient().getAllPets(user.getToken(),
-                        user.getUsername()));
+                userPets.set(ServiceLocator.getInstance().getPetManagerClient().getAllPets(user.getToken(), user.getUsername()));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 129).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5643405)